uyuni-docs-helper

```diff
@@ -172,6 +172,20 @@ if [ -z ${PRODUCT} ]; then
     print_incorrect_syntax "Product (-p) is mandatory"
 fi
 
+# Validate product selection
+VALID_PRODUCTS=("uyuni" "mlm" "suma")
+if [[ ! " ${VALID_PRODUCTS[@]} " =~ " ${PRODUCT} " ]]; then
+    print_error "Invalid product. Allowed: uyuni, mlm, suma"
+    exit 1
+fi
+
+# Check if the build fails
```

Comment:
This doesn't make sense here: `if [ $? -ne 0 ]; then` evaluates if the last command returned an error. IIUC you want this to show an error if calling `configure` fails, but here we're not calling, and `configure` (or `make configure` should produce this error.